docs/source/development/commands_reference.md

```diff
@@ -279,6 +279,10 @@ def run(
 kedro build-reqs
 ```
 
+___
+_This command will be deprecated from Kedro version 0.19.0._
+___
```

Comment:
We don't really use this style of formatting anywhere else. Perhaps just add this as a {note} or with `>` indenting. 

See https://github.com/kedro-org/kedro/blob/main/docs/source/tutorial/create_pipelines.md for example of "notes"

Reply:
I understand, I fixed it to use note syntax in the latest commits.